pom.xml

```diff
@@ -554,31 +553,31 @@
         <dependency>
             <groupId>org.junit.vintage</groupId>
             <artifactId>junit-vintage-engine</artifactId>
-            <version>5.11.0</version>
+            <version>5.11.3</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.platform</groupId>
             <artifactId>junit-platform-suite</artifactId>
-            <version>1.11.1</version>
+            <version>1.11.3</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
-            <version>${jupiter.version}</version>
+            <version>5.11.3</version>
```

Comment:
By removing this we can more easily automatically manage the latest versions of dependencies using the command `mvn versions:use-latest-versions`, otherwise having a indirection here with a variable makes this command update everything except for that version in the `pom.xml`. I agree that there is a trade off here where using a property like this forces all the libraries to be the same version.

Reply:
Yea the above command goes out to mvn central, figures out all the latest versions, then modifies the pom.xml to be latest greatest. This seems like the fastest way for us to on occasion update dependencies to latest.